uniflow/flow/config.py

```diff
@@ -186,6 +186,93 @@ def check_labels_in_label2score(self) -> bool:
         return True
 
 
+@dataclass
+class RaterForGeneratedAnswerConfig(RaterConfig):
+    """Rater classification Config Class."""
+
+    flow_name: str = "RaterForGeneratedAnswerFlow"
+    model_config: ModelConfig = OpenAIModelConfig()
+    label2score: Dict[str, float] = field(
+        default_factory=lambda: {
+            "Strong accept": 2.0,
+            "Accept": 1.0,
+            "Equivalent": 0.0,
+            "Reject": -1.0,
+            "Strong reject": -2.0,
+        }
+    )
+    # NOTE: This flow seems very sensitive to the choice of prompt.
+    # For a more stable performance, prompt should be improved.
+    guided_prompt_template: GuidedPrompt = GuidedPrompt(
+        instruction="""Rate the generated answer compared to the grounding answer to the question. Accept means the generated answer is better than the grounding answer and reject means worse.
+        Follow the format of the examples below to include context, question, grounding answer, generated answer and label in the response.
+        The response should not include examples in the prompt.""",
+        examples=[
+            Context(
+                context="Basic operating system features were developed in the 1950s, such as resident monitor functions that could automatically run different programs in succession to speed up processing.",
+                question="When were basic operating system features developed?",
+                grounding_answer="Basic operating system features were developed in the 1980s.",
+                generated_anser="Basic operating system features were developed in the 1950s",
```

Comment:
The `generated_answer` and `grounding_answer` are supposed to be different, so the LLM can compare the difference.

Reply:
`generated_anser` typo

Reply:
Actually the grounding_answer is "1980s" and generated_answer is "1950s", but I do think this example may not be too appropriate.